js/exporter/jspdf/v3/rows_splitting.js

```diff
@@ -1,15 +1,30 @@
 import { isDefined } from '../../../core/utils/type';
 import { roundToThreeDecimals } from './draw_utils';
+import { getPageWidth } from './pdf_utils_v3';
+
+function splitRectsByPages(doc, rowsInfo, options, onSeparateRectHorizontally) {
+    const rects = [].concat.apply([],
```

Comment:
In the method arguments, we no longer pass an **rects[]** and do not return **rects[]**.
Let's rename the method name?  splitRectsByPages => splitRowsByPages ?

Reply:
Yep, great idea.
Fixed